### PR TITLE
{math}[dummy/] MATLAB v2018b

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MATLAB/MATLAB-R2018b.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name = 'MATLAB'
+version = 'R2018b'
```
Comment (boegel) [inline easybuild/easyconfigs/m/MATLAB/MATLAB-R2018b.eb]: @vanzod In previous `MATLAB` easyconfig files, we've never used the `R` in the version, no need to start doing that?

Comment (vanzod) [reply easybuild/easyconfigs/m/MATLAB/MATLAB-R2018b.eb]: Fixed

File: easybuild/easyconfigs/m/MATLAB/MATLAB-2018b.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# be sure to copy both DVD content to the SAME directory, 
+# including the hidden files, especially .dvd1 and .dvd2 
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('Java', '1.8')]
+
+license_server = 'my-license-server'
+license_server_port = '1234'
+key = '00000-00000-00000-00000-00000-00000-00000-00000-00000-00000-00000-00000'
```
Comment (boegel) [inline easybuild/easyconfigs/m/MATLAB/MATLAB-2018b.eb]: @vanzod These should be commented out, so the MATLAB easyblock can pick up on `$EB_MATLAB_KEY` & co...

Comment (verdurin) [reply easybuild/easyconfigs/m/MATLAB/MATLAB-2018b.eb]: Yes, and to cover the cases where server-specific licenses are being used.

Comment (vanzod) [reply easybuild/easyconfigs/m/MATLAB/MATLAB-2018b.eb]: Done. I also added a short comment for clarity

File: easybuild/easyconfigs/m/MATLAB/MATLAB-2018b.eb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('Java', '1.8')]
+
+# Uncomment and modify the following variables if needed 
+# for installation with floating license server
+#license_server = 'my-license-server'
+#license_server_port = '1234'
+#key = '00000-00000-00000-00000-00000-00000-00000-00000-00000-00000-00000-00000'
+#modextravars = {'LM_LICENSE_FILE': '%s@%s' % (license_server_port, license_server)}
```
Comment (boegel) [inline easybuild/easyconfigs/m/MATLAB/MATLAB-2018b.eb]: @vanzod You'll need to use `# modextravars` to make the style check happy... (i.e., add a space after the `#`)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar04.gastly.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/988f8b45c5c09d2f499b27005a530993 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**